keystore/keyloader/kms/kms_loader.go

```diff
@@ -93,14 +87,14 @@ func (loader *Loader) LoadMasterKeys() (encryption []byte, signature []byte, err
 	return keys.Encryption, keys.Signature, nil
 }
 
-func (loader *Loader) decryptWithKMSKey(keyID kms.KeyIdentifier) ([]byte, error) {
+func (loader *Loader) decryptWithKMSKey(keyID string) ([]byte, error) {
```

Comment:
lets use []byte type for identifier? most likely that we will use clientID and related things as keyIDs and much often it all will have []byte type.